### PR TITLE
add a helm template to release artifacts for ddesktop local deploys

### PR DESCRIPTION
**Pull request by molinamelendezj**

### CODE REVIEW

**Comment by molinamelendezj** [comment]
cc: @tybritten

**Comment by tybritten** [comment]
Do we definitely want to include the notebook part or just the base helm template?

**Comment by molinamelendezj** [comment]
> Do we definitely want to include the notebook part or just the base helm template?

I assumed so. was cool to see notebook setup and running locally. we can include it an improve any of this on future PRs but at least this template would be shipping on every release.

File: etc/deploy/dd/dd_notebook_base.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    matchLabels:
+      app: jupyter-notebook
+  template:
+    metadata:
+      labels:
+        app: jupyter-notebook
+    spec:
+      containers:
+        - name: pach-notebook
+          image: pachyderm/notebooks-user:vPACH_VERSION
```
Comment (djanicekpach) [inline etc/deploy/dd/dd_notebook_base.yaml:19]: should this change `vPACH_VERSION -> "v${PACH_VERSION}"`?

**Review by djanicekpach — APPROVED** [review]

File: etc/deploy/dd/dd_notebook_base.yaml (hunk 1 above)
Comment (molinamelendezj) [reply etc/deploy/dd/dd_notebook_base.yaml:19]: Naw, it's just a place holder, the `sed` replaces it at release time.

**Comment by chainlink** [comment]
Hey guys, just seeing this now, we shouldn't include the notebook portion (as it doesn't include storage, so the user will lose their work everytime) and the `sed` portion can be set with helm options (which is much safer)

**Comment by chainlink** [comment]
We also shouldn't call this the "docker desktop" option, since it doesn't work in docker desktop (because of the underlying PVC issue with DD)

**Pull request merged**